src/dmclock_client.h

```diff
@@ -122,9 +122,12 @@ namespace crimson {
 	} else {
 	  Counter delta =
 	    delta_counter - it->second.delta_prev_req;
+#ifdef USE_SEPARATED_RHO_CAL
```

Comment:
I understand the my_rho/my_delta change. I do not understand the USE_SEPARATED_RHO_CAL change. What benefit did it provide?